README.md

```diff
@@ -11,9 +11,9 @@ This project contains the official Go driver for the [ArangoDB database](https:/
 > **Warning**
 > V1 is deprecated and will not receive any new features. Please use [V2](v2) instead.
```

Comment:
Make clearer what v1 and v2 are, and since folder is called v2, I suggest we use "v2" instead of "V2" when referring to the driver version.

```suggestion
> Version 1 of the driver is deprecated and will not receive any new features.
> Please use version 2 ([v2](v2)) instead.
```